### DISCUSSION TITLE
Incorrect feedback force

### DISCUSSION THREAD

**Discussion by RickYang1** [InfinyTech3D/SofaUnity#81] [Help - Using SofaUnity] [answered]
Hi，
    I obtained the forces from the SofaMesh class, but when I printed it out, I found that the values of the three axes were completely equal.
![1718611889994](https://github.com/InfinyTech3D/SofaUnity/assets/142986134/01d26055-520d-4eec-b6ba-dbae758e2fc0)
    How should this be resolved.

**Comment by epernod** [comment]
Hi again,
sorry for the double answer, I'm processing the discussions one by one.

Which Data are you accessing? could you post your code here?
Best regards

**Comment by RickYang1** [comment]
Hello @epernod,
    Thank you for your reply！The array named “forces” in the SofaMesh class that I accessed，as shown in the following figure:
![image](https://github.com/InfinyTech3D/SofaUnity/assets/142986134/64465add-e8eb-4670-aa3c-0931fef9b94b)
Line 375 is my code, the rest are the original code.
Best regards.

**Reply by epernod** [reply, marked as answer]
Ok, the forces Data in Mesh correspond to the internal + external forces applied to each point.
To have a better visualization you should display each force vector3 ith at the position ith
I will check why x,y,z have the same values. It might be a bug from the API.

thank you
